ro_help/ro_help/settings/dev.py

```diff
@@ -8,7 +8,7 @@
 
 AUTH_PASSWORD_VALIDATORS = []
 
-# EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
```

Comment:
This might need to be commented out again.

Reply:
we should fix it the right way :)